src/iotconnect_telemetry.c

```diff
@@ -158,11 +133,11 @@ bool iotcl_telemetry_add_with_epoch_time(IotclMessageHandle message, time_t time
 
 bool iotcl_telemetry_add_with_iso_time(IotclMessageHandle message, const char *time) {
     if (!message) return false;
-    cJSON *const telemetry_object = setup_telemetry_object(message);
-    if (!telemetry_object) return false;
-    if (!cJSON_AddStringToObject(telemetry_object, "dt", time)) return false;
-    if (!cJSON_HasObjectItem(message->root_value, "t")) {
-        if (!cJSON_AddStringToObject(message->root_value, "t", time)) return false;
+    cJSON *const telemetry_object = setup_telemetry_object(message, time);
+    char *sensors_data = cJSON_Print(telemetry_object);
```

Comment:
Is "char *sensors_data = cJSON_Print(telemetry_object);" unused -- if so, then remove